packages/voyager-server/src/voyagerResolver.ts

```diff
@@ -29,13 +29,11 @@ function voyagerResolverPartial (config: CompleteVoyagerConfig): ResolverWrapper
           }
 
           const timeTook = Date.now() - resolverStartTime
-          metrics.updateResolverMetrics(info, timeTook)
+          metrics.updateResolverMetrics(true, context, info, timeTook)
           auditLogger.logResolverCompletion('', true, obj, args, context, info)
         } catch (error) {
-          // we only publish time in success. const timeTook
-          // NOPE: const timeTook = Date.now() - resolverStartTime
-          // NOPE: updateResolverMetrics(info, timeTook)
-          // but, we audit log in case of failure too
+          const timeTook = Date.now() - resolverStartTime
+          metrics.updateResolverMetrics(false, context, info, timeTook)
```

Comment:
I wanted to change the behavior here. Previously, when resolver failed we didn't record that and Prometheus didn't receive that. Now, after introduction of `success` field, we can do this. Prom will now get the info but it will also get the success/fail.